grass7/imagery/i.gabor/i.gabor.html

```diff
@@ -5,6 +5,9 @@ <h2>DESCRIPTION</h2>
 filter bank and raster map.
 The 2 dimensional Gabor filter is an orientation sensitive filter which mimics
 the cells within the human visual cortex.
+The module can either produce individual rasters for with the prefix of
```

Comment:
`for with the prefix` => `with the prefix`

I believe using the basename as prefix and appending filter information would be better than the other way around.

`basename_windowsize_orientation_waelength_offset_aspect` would be better than `windowsize_orientation_wavelength_offset_aspect_basename` for organizing outputs.

Reply:
That makes much more sense. ``windowsize_orientation_wavelength_offset_aspect_basename`` would get messy quickly.

Reply:
@HuidaeCho This can be resolved now, or ?

Reply:
> @HuidaeCho This can be resolved now, or ?

It is already resolved. `output=` is now prefix.

Let me add some final touches to the manual.